node-infrastructure/run-a-node/polkadot-hub-rpc.md

```diff
@@ -21,22 +21,23 @@ Through the Polkadot SDK node RPC (WebSocket port 9944, HTTP port 9933), your no
 
 RPC nodes serving production traffic require robust hardware. The following should be considered the minimum standard to effectively operate an RPC node:
 
-- **CPU**: 8+ cores (16+ cores for high traffic)
-- **Memory**: 64 GB RAM minimum (128 GB recommended for high traffic)
+- **CPU**: 8+ cores; 16+ cores for high traffic
+- **Memory**: 64 GB RAM minimum; 128 GB recommended for high traffic
 - **Storage**:
     - **Archive node**: ~1.2 TB NVMe SSD total (~392 GB for Asset Hub archive + ~822 GB for relay chain pruned snapshot)
+    <!-- TODO-ERIN: this means ~1.4 TB? 👇 -->
     - **Pruned node**: 200+ GB NVMe SSD (with pruning enabled for both parachain and relay chain)
```

Comment:
@brunopgalvao same question here as [this](https://github.com/polkadot-developers/polkadot-docs/pull/1246#discussion_r2608698280)